src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
Thanks for the research done so far. I can still go forward with merging if we can't find a definitive source, but I want to make sure that we did due-diligence before adding in types that don't have any real concrete source.

If there is ample evidence that both are necessary, we could even add both.